lib/scheduler.js

```diff
@@ -384,6 +407,16 @@ class Scheduler {
             return null;
         }
     }
+
+    _isAstro(ev) {
```

Comment:
"is" methods normally return true or false ... maybe call it "getAstro" or  "findAstroEntry" or such?

Reply:
thought about this, originally i only wanted true or false but then needed more and forgot about naming. Thank you very much for teaching me Best Practices, appreciate that. Though i wanted you to have less work, not more :-) new commits to come later tonight...

Reply:
All fine, I know much code in adapter might notbe optimal ... but we try to have new code/changes to be "Better" :-)

Reply:
tests fail because testScheduler.js has missing suncalc at new Scheduler(null, Date). could you please fix this so tests will pass?
i was planning to add some test cases for astro anyway, however i failed, mainly because i have no clue how that all works together :-)
* could you give me a hint please on how to ONLY run a test for testScheduler.js? 
  * Can this be done by removing all other tests from the test folder temporarily?
* can this then be a separate PR, since i need some time for this

thx in advance

Reply:
all fine